data_tools/firebase_client_wrapper.py

```diff
@@ -45,16 +46,18 @@ def get_message_ids(dataset_id):
         ids.append(message.id)
     return ids
 
-# This is a much faster way of doing an entire dataset read than repeated get_message calls
+# This is a much faster way of reading an entire dataset read than repeated get_message calls
 def get_all_messages(dataset_id):
     messages = []
     for message in client.collection(u'datasets/{}/messages'.format(dataset_id)).get():
         messages.append(message.to_dict())
     return messages
 
-
 def get_message(dataset_id, message_id):
     return client.document(u'datasets/{}/messages/{}'.format(dataset_id, message_id)).get().to_dict()
 
 def get_message_ref(dataset_id, message_id):
     return client.document(u'datasets/{}/messages/{}'.format(dataset_id, message_id))
+
+def push_coding_status(coding_status):
+    client.document(u'metrics/coda').set(coding_status)
```

Comment:
Can you also add a newline at the end of this file? Github shows a warning for it.

Reply:
done